### ISSUE TITLE
Feed does not show all tweets

### ISSUE THREAD

**Issue by tominer** [j-fbriere/squawker#214]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/j-fbriere/squawker/releases).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/j-fbriere/squawker/issues) or [closed](https://github.com/j-fbriere/squawker/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug, that only occurs in Squawker

### Affected version

3.7.3

### Steps to reproduce the bug

1. Add @ThailandMedicaX to supscriptions
2. Look for the tweets in the Squawker Feed and compare them with the tweets of that account - there are multiple tweets missing in the Feed

### Expected behavior

Showing all tweets from the account ThailandMedicaX in the Feed.

### Affected Android/Custom ROM version

A13 eos

**Comment by beerninja** [comment]
Same problem here.  Nearly every subscription is missing tweets in feed but the tweets show up fine if I view the individual accounts one at a time.

**Comment by j-fbriere** [comment]
I've noticed that if I don't check the feed regularly this type of behavior appears sometime. 
It might be due to the fact that the Twitter/X API for the feed request is very rate limited.

**Comment by Bazzz66** [comment]
Noticed the same issue, from the subscriptions all tweets are visible. From the feed quite a lot of tweets are not visible (I'm assuming that this is not a setting of the app)

**Comment by dailylhama** [comment]
Can confirm for 3.8.1.

This is what actually keeps me from using Squawker. I don't know if it is oft any relevance, but this also happens in Quacker.

**Comment by tominer** [comment]
Perhaps its possible to add a feature that collects the news feeds from the account site rather than with the actual api.

**Comment by j-fbriere** [comment]
Can someone confirms (or negates) from his/her experience that by refreshing the feed more regularly (every 30 min for instance) then there is no tweet missing?

**Comment by tominer** [comment]
No, I cant confirm this. Tried today refreshing every 30 minutes, but the posts are still missing.

**Comment by fjelsten** [comment]
I am also seeing this.

If you lick on a user's name, alle tweets are available.

Guess: If the number of tweets that needs fetching is larger than X, not all old stuff is fetched. As someone suggested, would it be possible to check each subscribed user to see if tweets are missing?

Or, have a button, the the bottom of the fetched feed saying "fetch older" or something like that?

Love the app (much better than the official).

**Comment by sshaikh** [comment]
I started a discussion for what I think is this behaviour here: https://github.com/j-fbriere/squawker/discussions/431.

I think the official X app has the "more feeds" function when you come back after some time. I think I'd prefer Squawker to automatically update the feed 3-4 hours after the last refresh though.
